include/zephyr/drivers/video-controls.h

```diff
@@ -161,6 +161,11 @@ enum video_power_line_frequency {
  * @}
  */
 
+struct video_control {
```

Comment:
these four lines also don't seem to need their own commit, just add it in the commit adding the rest of the video controls code